spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -58,6 +58,16 @@
       expect(run_engine(engine_conf)).to be_empty
   end
 
+  it "does not report the same line for multiple issues" do
+    create_source_file("dup.jsx", <<-EOJSX)
+          <a className='button button-primary full' href='#' onClick={this.onSubmit.bind(this)}>Login</a>
+    EOJSX
+
+    result = run_engine(engine_conf).strip
+    issues = result.split("\0")
+    expect(issues.length).to eq 1
```

Comment:
@BlakeWilliams can you explain why this is 1? Is it because we set the mass to 1 for the tests? I wouldn't expect a line to be reported unless there was another instance of it